manifest_staging/charts/gatekeeper/values.yaml

```diff
@@ -7,7 +7,9 @@ image:
   repository: quay.io/open-policy-agent/gatekeeper
   release: v3.1.0-beta.8
   pullPolicy: IfNotPresent
-nodeSelector: {}
+nodeSelector: {
```

Comment:
@sozercan 

I forgot, these get clobbered now. You need to change the source file in the helmify directory.

Reply:
You  should also update the README

Reply:
thanks @maxsmythe, opened #629